tests/ja4-rules/suricata.yaml

```diff
@@ -0,0 +1,28 @@
+%YAML 1.1
+---
+
+outputs:
+  - eve-log:
+      enabled: yes
+      filetype: regular #regular|syslog|unix_dgram|unix_stream|redis
+      filename: eve.json
+      types:
+        - alert:
+            payload: no
+            payload-buffer-size: 4kb
+            payload-printable: no
+            packet: no
+            metadata: yes
+        - tls:
+            extended: yes
+        - quic
+
+app-layer:
+  protocols:
+    tls:
+      enabled: yes
+      detection-ports:
+        dp: 443
+      ja4-fingerprints: yes
```

Comment:
If we just need this, it can be a command line argument `--set`

Reply:
True. Having a `suricata.yaml` also allows me to control the types that go into the EVE etc. It gives me a better overview of what is done apart from the defaults. If you insist I can change it, but I didn't find it too intrusive to just include a full config file.

Reply:
ok for me